src/ZipStream.php

```diff
@@ -21,12 +20,16 @@
  * @author Jesse Donat <donatj@gmail.com>
  * @copyright Copyright (C) 2014 Jesse Donat <donatj@gmail.com>
  *
- * @license https://raw.githubusercontent.com/maennchen/ZipStream-PHP/master/LICENCE
+ * @author Nik Barham <github@brokencube.co.uk>
+ * @copyright Copyright (C) 2016 Nik Barham <github@brokencube.co.uk>
+ *
+ * @license https://raw.githubusercontent.com/brokencube/ZipStream64/master/LICENCE
```

Comment:
The license should remain in the original location.

Reply:
Sorry, fixed.